efi-log.sh

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (C) 2024  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Paweł Poławski <ppolawsk@redhat.com>
+
+# Ignore unused variable parsed out by tooling scripts as test tags metadata
+# shellcheck disable=SC2034
+TESTTYPE="efi log coverage"
```

Comment:
- I think the test does not belong to coverage (it is ~30 selected tests).
- Should it be rather `logs` than `log` (we already have tests in the former category)
- Is it of `efi` type? I don't see it runs on uefi in the tests (like for example in https://github.com/rhinstaller/kickstart-tests/blob/54ab7e2db056141791784712907ff467c7b55a3e/reboot-uefi.sh#L24). Is it required for the test to run on uefi? If so then i'd rather use `uefi` both as the type and in the name, again we already have 2 similar tests.

Reply:
Done